rts/Game/Game.cpp

```diff
@@ -1643,6 +1653,8 @@ void CGame::SimFrame() {
 
 	good_fpu_control_registers("CGame::SimFrame");
 
+	FrameMarkStart(tracingSimFrameName);
```

Comment:
I'd assume if you place `"SimFrame"` here it should also work via string pooling.

Reply:
I don't think stirng pooling is guaranteed by the standard, so I don't want to depend on that. See tracy doc 3.1.2, they also reference standard section from there.